cmd/conduit/internal/print_utils.go

```diff
@@ -195,3 +195,17 @@ func formatValidations(v []*configv1.Validation) string {
 	}
 	return result.String()
 }
+
+// ProcessorTypeToString returns a human-readable string from a processor parent type
+func ProcessorTypeToString(processorParentType apiv1.Processor_Parent_Type) string {
```

Comment:
The word "type" is confusing to me here, how about we call this "parent" instead, like it is internally named? So that we would display `Parent: connector (conn-id-foo)`.

Reply:
@lovromazgon that works

Reply:
https://github.com/ConduitIO/conduit/pull/2097/commits/1d381aee2095cdca234e1275ef4d3acf4e6cd62d